web/models.py

```
@@ -1144,3 +1144,33 @@ class Meta:
+    is_public = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ["-issue_date"]
+
+    def __str__(self):
+        return f"{self.title} - {self.user.username}"
+
+    def get_absolute_url(self):
+        return reverse("certificate_detail", kwargs={"uuid": self.uuid})
```

Ruff (0.8.2)

Undefined name `reverse`

(F821)